src/Microsoft.IdentityModel.Logging/TelemetryInstrumentation.cs

```diff
@@ -0,0 +1,57 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Diagnostics;
+
+namespace Microsoft.IdentityModel.Logging
+{
+    internal class TelemetryInstrumentation : ITelemetryInstrumentation
+    {
+        static TagList ClientVerTagList = new TagList()
+        {
+            { TelemetryConstants.IdentityModelVersionTag, IdentityModelTelemetryUtil.ClientVer }
+        };
+
+        public void IncrementOperationCounter(string operationStatus)
+        {
+            var tagList = new TagList()
+            {
+                { TelemetryConstants.IdentityModelVersionTag, IdentityModelTelemetryUtil.ClientVer },
+                { TelemetryConstants.OperationStatusTag, operationStatus }
+            };
+
+            IdentityModelTelemetry.IncrementOperationCounter(tagList);
+        }
+
+        public void IncrementOperationCounter(string operationStatus, Exception exception)
+        {
+            var tagList = new TagList()
+            {
+                { TelemetryConstants.IdentityModelVersionTag, IdentityModelTelemetryUtil.ClientVer },
+                { TelemetryConstants.OperationStatusTag, operationStatus },
+                { TelemetryConstants.ExceptionTypeTag, exception.GetType().ToString() }
```

Comment:
Can any of these be null?

Reply:
No, GetType() will at the very least return type `Object` (Except in very specific cases that do not apply here)